Code/GraphMol/FileParsers/test1.cpp

```diff
@@ -4229,6 +4230,133 @@ void testRCSBSdf() {
   delete mol;
 }
 
+// BAD PDB Ligand with CHG line too long (>8) and right and mid-justified symbols
+const std::string molblock_chg = "2D1G_DVT_B_2001\n" \
```

Comment:
For testing it doesn't make a difference, and honestly it's easier to make the sdf file without the alignment inside the function :)

I moved it in this round .